libcontainer/configs/config.go

```diff
@@ -33,9 +33,10 @@ type Seccomp struct {
 type Action int
 
 const (
-	Kill Action = iota - 4
```

Comment:
why not do it like this ?
<pre><code>
const (
	Kill Action = iota
	Errno
	Trap
	Allow
	Trace
)
<pre><code>